spec/application.js

```diff
@@ -25,6 +26,22 @@ const decisionSchema = {
   },
   additionalProperties: false
 }
+const applicantSchema = {
+  type: 'object',
+  required: ['preferredName', 'legalName', 'aliveInterval', 'city', 'country'],
+  properties: {
+    // required fields
+    preferredName: { type: 'string' },
+    legalName: { type: 'string' },
+    aliveInterval: { type: 'string' },
+    city: { type: 'string' },
+    country: { type: 'string' }
+
+    // optional fields
+    // TODO: add optional fields, ensure these fields are nullable
```

Comment:
TODO

Reply:
I think it would be good to include avatarImage.

We're shooting for something where the kaitiaki can use this to build an in tribe profile for the applicant.

I think some of these fields should be optional? Probably the image and preferredName?

Reply:
Still waiting to hear back from Ben about which attrs to include, the ones i've added so far are the required fields. Will update this PR when i get his response. Deff agree about the avatarImage

Reply:
Using profileId instead of applicant object, resolving this conversation